content/tutorial/02-advanced-svelte/01-motion/02-springs/app-a/src/lib/App.svelte

```diff
@@ -11,6 +11,7 @@
 	}}
 	on:mousedown={() => size.set(30)}
 	on:mouseup={() => size.set(10)}
+	role="none"
```

Comment:
i think this is _marginally_ less weird

```suggestion
	role="presentation"
```